src/examples/poi-crosschecker/poi-crosschecker.ts

```diff
@@ -20,10 +20,10 @@ const RADIO_PAYLOAD_TYPES = [
 const run = async () => {
   const clientManager = new ClientManager({
     operatorPrivateKey: process.env.RADIO_OPERATOR_PRIVATE_KEY,
-    ethNodeUrl: process.env.ETH_NODE,
+    ethNodeUrl: `http://${process.env.ETH_NODE}`,
```

Comment:
Isn't it better to leave it for the .env file to specify the full url? that way Infura can be used too

Reply:
this is more consistent with the other endpoints, I can let other ones also specify the full url

Reply:
Yeah that sounds good, let's err on the side of more freedom of the user